drivers/Makefile.dep

```diff
@@ -199,6 +199,12 @@ ifneq (,$(filter hts221,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter il3829,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_spi
+  FEATURES_REQUIRED += periph_gpio
+  USEMODULE += spi_display
```

Comment:
Shouldn't we call it spi_epd, spi_epaper_display or something similar? spi_display is generic. This could also be an OLED or LCD.

Reply:
`spi_display` was very generic (and pretty useless), I have now reworked it to `epd_bw_spi` for the specific class of displays.